compiler/stable_mir/src/lib.rs

```diff
@@ -196,13 +197,19 @@ pub trait Context {
     /// Find a crate with the given name.
     fn find_crates(&self, name: &str) -> Vec<Crate>;
 
-    /// Prints the name of given `DefId`
+    /// Returns the name of given `DefId`
     fn name_of_def_id(&self, def_id: DefId) -> String;
 
-    /// Prints a human readable form of `Span`
+    /// Returns printable, human readable form of `Span`
```

Comment:
Good catch! Can you rename the method as well since we are not printing it? Maybe span_display or span_to_string.